src/bundle/SystemInfo/Collector/IbexaSystemInfoCollector.php

```diff
@@ -146,7 +146,7 @@ class IbexaSystemInfoCollector implements SystemInfoCollector
     private $kernelProjectDir;
 
     /**
-     * @param \Ibexa\Bundle\SystemInfo\SystemInfo\Collector\JsonComposerLockSystemInfoCollector|\Ibexa\Bundle\SystemInfo\SystemInfo\Collector\SystemInfoCollector $composerCollector
+     * @param \Ibexa\Bundle\SystemInfo\SystemInfo\Collector\JsonComposerLockSystemInfoCollector|SystemInfoCollector $composerCollector
```

Comment:
FQCN was removed by accident.
```suggestion
     * @param \Ibexa\Bundle\SystemInfo\SystemInfo\Collector\JsonComposerLockSystemInfoCollector|\Ibexa\Bundle\SystemInfo\SystemInfo\Collector\SystemInfoCollector $composerCollector
```

Reply:
I wonder if that even make much sense to have `\Ibexa\Bundle\SystemInfo\SystemInfo\Collector\JsonComposerLockSystemInfoCollector` here, as it is implementation of `\Ibexa\Bundle\SystemInfo\SystemInfo\Collector\SystemInfoCollector`

Reply:
If one is an interface that the other implements, then implementation can be removed from declaration.